packages/e2e.generated.api-namespace.dep/src/generatedNoCheck/OntologyMetadata.ts

```diff
@@ -1,2 +1,4 @@
 export type $ExpectedClientVersion = '2.0.0';
 export const $osdkMetadata = { extraUserAgent: 'typescript-sdk/dev osdk-cli/dev' };
+
+export const $ontologyRid = 'ri.ontology.main.ontology.dep';
```

Comment:
We no longer have a good way to know if we are generating a shared ontology package vs a "local' package so we have to always include this.

Reply:
its in the FAQ for sure